### PR TITLE
[IE CLDNN] Eltwise fsv4 opt and bugfix 

### PR DESCRIPTION
**Pull request by byungilm**
### Details:
 - optimzing eltwise fsv4 layout

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+    dispatchData.lws= GetLocalWorkGroupSizes(dispatchData.gws, params.engineInfo);
+    dispatchData.lws[1] = 1;
+    dispatchData.lws[2] = 1;
+
+    return dispatchData;
+}
+
+// Local
+static std::vector<size_t> GetLocalWorkGroupSizes(std::vector<size_t> gws, const EngineInfo& info) {
```
Comment (hyunback) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp:262]: Is this same func in kernel_selector_utils.cpp (GetOptimalLocalWorkGroupSizes)

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    const auto& ewParams = static_cast<const eltwise_params&>(params);
+
+    const auto& output = ewParams.output;
+    const auto count = output.PhysicalSize();
+
+    if (count % vec_size != 0)
+        return false;
+
+    for (size_t i = 0; i < ewParams.inputs.size(); i++) {
+        if (ewParams.inputs[i].GetLayout() != DataLayout::b_fs_yx_fsv4) {
```
Comment (hyunback) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp:81]: How about add one more condition?
if (ewParams.inputs[i].GetLayout() != DataLayout::b_fs_yx_fsv4 &&
       && ewParams.inputs[i].LogicalSize() != 1)

ex> two input tensor: b_fs_yx_fsb4(4, 2, 2, 2), bfyx(1, 1, 1, 1)
selected kernel:  ref_kernel  -> fsv4_kernel

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+    if (!params.fused_ops.empty()) {
+        kernel_selector::Datatype input_dt = GetAccumulatorType(params);
+        std::vector<std::string> idx_order = {"b", "f_block*4", "y", "x"};
+        FusedOpsConfiguration conf = {"", idx_order, "res", input_dt, (size_t)vec_size};
+        conf.vec_axis = Tensor::DataChannelName::FEATURE;
+        jit.Merge(MakeFusedOpsJitConstants(params, {conf}));
+    }
+
+    jit.AddConstant(MakeJitConstant("ELTWISE_BROADCAST", params.broadcast));
```
Comment (hyunback) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp:240]: With below code, we can avoid unnecessary calculating safe idx for tensor(1, 1, 1, 1) case.
    if (params.broadcast) {
        bool need_idx_safe = true;
        for (size_t i = 0; i < params.inputs.size(); i++) {
            if ((params.inputs[i].LogicalSize() == 1)) {
                    need_idx_safe = false;
                    break;
            }
        }
        if (need_idx_safe)
            jit.AddConstant(MakeJitConstant("ELTWISE_BROADCAST", params.broadcast));
    }

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp (hunk 1 above)
Comment (byungilm) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp:262]: Thank you. Removed.

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp (hunk 2 above)
Comment (byungilm) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp:81]: Thanks. Added.

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp (hunk 3 above)
Comment (byungilm) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp]: Applied.

Comment (byungilm) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp]: Additionally I modified the logic to generate INPUT Jit when LogicalSize is 1.

**Comment by byungilm** [comment]
duplicated to https://github.com/openvinotoolkit/openvino/pull/5104

**Pull request closed without merging**